src/Pond.tsx

```diff
@@ -31,25 +31,79 @@ type PondProps = {
   /** Component to show during the connect (very shortly) */
   loadComponent?: JSX.Element
   /** Error callback the the pond is not able to reach actyxOS locally */
-  onError?: (error: unknown) => void | JSX.Element
-  /** optional url to overwrite to local connection and connect to an other peer (default: `ws://localhost:4243/store_api`) */
-  url?: string
-  /** Hook, when the connection to the store is closed */
-  onStoreConnectionClosed?: () => void
-  /** Advanced configuration options for the Pond. */
-  pondOptions?: PondOptions
+  onError: (error: unknown) => void | JSX.Element
+  /**
+   * Optionally change the Actyx endpoint the Pond connects to
+   * and provide an error handler if the connection to Actyx is dropped.
+   * Defaults to `{}`
+   * */
+  connectionOpts?: ActyxOpts
+  /** 
+   * Advanced Pond configuration options.
+   * Defaults to `{}`
+   *  */
+  opts?: PondOptions
+  /**
+   * Manifest describing an Actyx application. Used for authorizing API access.
+   * Use `com.example.<somestring>` as `appId` for testing and development purposes,
+   * so you don't require a signed certificate.
+   * 
+   * Example:
+   * ```
+   * {
+   *   appId: 'com.example.react-pond-example',
+   *   displayName: 'React Pond Example',
+   *   version: '0.0.1'
+   * }
+   * ```
+   * */
+  manifest: AppManifest
 }
 
+
 /** @internal */
 let singletonPond: PondType | undefined = undefined
 
 /**
  * Top level component to initialize the pond
  *
- * ## Example:
+ * ## Minimal example:
  * ```js
  * ReactDOM.render(
- *   <Pond>
+ *   <Pond 
+ *     onError={onError}
+ *     manifest={{
+ *       appId: 'io.actyx.react-pond-example',
+ *       displayName: 'React Pond Example',
+ *       version: '0.0.1',
+ *       signature: 'v2tz...JBPT3/'
+ *     }}
+ *   >
+ *     <App />
+ *   </Pond>,
+ *   document.getElementById('root')
+ * )
+ * ```
+ * 
+ * ## Complete example:
+ * ```js
+ * ReactDOM.render(
+ *   <Pond onError={onError}
```

Comment:
linefeed is somehow broken here.
The evel linter will not help you here :-) :exploding_head:

Reply:
I don't see it. What do you mean?